Lib/test/test_class.py

```diff
@@ -3,7 +3,7 @@
 import unittest
 
 from test import test_support
-
+    
```

Comment:
A few blank line additions have crept into this commit, which might make merge conflicts a bit more likely later. Probably worth putting things back as they were.